File: jambeth/jambeth-cache-datachange/src/main/java/de/osthus/ambeth/cache/CacheValueAndPrivilege.java
package de.osthus.ambeth.cache;

import de.osthus.ambeth.cache.rootcachevalue.RootCacheValue;
import de.osthus.ambeth.privilege.model.IPrivilege;

public class CacheValueAndPrivilege
{
	public final RootCacheValue cacheValue;

	public final IPrivilege privilege;

	public CacheValueAndPrivilege(RootCacheValue cacheValue, IPrivilege privilege)
	{
		this.cacheValue = cacheValue;
		this.privilege = privilege;
	}
}

File: ambeth/Ambeth.CacheDataChange/ambeth/cache/CacheValueAndPrivilege.cs
using De.Osthus.Ambeth.Cache.Rootcachevalue;
using De.Osthus.Ambeth.Privilege.Model;

namespace De.Osthus.Ambeth.Cache
{
    public class CacheValueAndPrivilege
    {
        public readonly RootCacheValue cacheValue;

        public readonly IPrivilege privilege;

        public CacheValueAndPrivilege(RootCacheValue cacheValue, IPrivilege privilege)
        {
            this.cacheValue = cacheValue;
            this.privilege = privilege;
        }
    }
}